utils/utils.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import (
+	"os"
+	"strconv"
+	"strings"
+)
+
+// Retrieves the specified environmental variable with type of Boolean as string, set it with defualt variable if not exist or unacceptable format
```

Comment:
nit: maybe "Parse environment variable and return boolean representation of string, or default value if environment variable is unset"